lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -779,72 +782,65 @@ sub _failed_check {
     my $hiddens = {
        check => $check->name,
 verify_check => md5_hex($check->test_query),
-    database => $request->{database}
+    database => $request->{database},
     };
+    my @skip_keys = grep /^skip_/, keys %$request;
+    $hiddens->{@skip_keys} = $request->{@skip_keys};
 
     my $rows = [];
     while (my $row = $sth->fetchrow_hashref('NAME_lc')) {
-      my $id = $row->{$check->{id_column}};
+      my $count = 1+scalar(@$rows);
+      #TODO: A '' will fail with invalid input syntax in fix_tests - YL
+      my $id = join(',',map { $row->{$_} // '' } @{$check->id_columns});
       for my $column (@{$check->columns // []}) {
         my $selectable_value = $selectable_values{$column};
+        my $name = $column . '_' . $count;
```

Comment:
This makes `$name` to be the same value for all cells in a single column, right?

Reply:
No, `$count` changes everytime we push a line in `$rows`